src/client/resources/assets/snapper/lang/pt_br.json

```diff
@@ -0,0 +1,42 @@
+{
+  "key.snapper.panorama": "Faça Capturas de Tela panorâmicas",
+  "key.snapper.huge": "Faça Captura de Tela enorme",
+  "key.snapper.screenshot_menu": "Exibir Capturas de Tela",
+  "key.snapper.recent": "Abrir a última Captura de Tela",
+  "menu.snapper.screenshotmenu": "Capturas de Tela",
+  "menu.snapper.viewermenu": "Visualizador",
+  "menu.snapper.panorama": "Exibir Panorama",
+  "button.snapper.folder": "Abrir Pasta",
+  "button.snapper.panorama.tooltip": "Visualizar o panorama atual armazenado.",
```

Comment:
Should be "Visualizar o panorama armazenado atualmente.". This would mean "View the stored current panorama" vs. "View the currently stored panorama"

Reply:
It comes across more correctly when it's worded properly.